src/lib/image_hash/average_hash.h

```diff
@@ -51,6 +51,10 @@ namespace image_hash
 				uint64_t mval = (*hax) & 0x101010101ULL;
 				const uint8_t* cv = reinterpret_cast<const uint8_t*>(&mval);
 				uint8_t val = cv[0] << 4 | cv[1] << 3 | cv[2] << 2 | cv[3] << 1 | cv[4];
+				// TODO:
+				/*if (bigEndian)
```

Comment:
Everything I'm targeting right now is littleendian, but this little optimization/hack will need some extra care for other platforms...